package.json

```
@@ -21,10 +21,10 @@
     "ios-standalone": "./scripts/set-pusher-suffix.sh && STANDALONE_NEW_DOT=true ./scripts/run-build.sh --ios",
     "pod-install": "./scripts/pod-install.sh",
     "pod-install-standalone": "STANDALONE_NEW_DOT=true ./scripts/pod-install.sh",
-    "ipad": "concurrently \"./scripts/run-build.sh --ipad\"",
-    "ipad-standalone": "concurrently \"STANDALONE_NEW_DOT=true ./scripts/run-build.sh --ipad\"",
-    "ipad-sm": "concurrently \"./scripts/run-build.sh --ipad-sm\"",
-    "ipad-sm-standalone": "concurrently \"STANDALONE_NEW_DOT=true ./scripts/run-build.sh --ipad-sm\"",
+    "ipad": "(npx react-native webpack-start & ./scripts/run-build.sh --ipad)",
+    "ipad-standalone": "(npx react-native webpack-start & STANDALONE_NEW_DOT=true ./scripts/run-build.sh --ipad)",
+    "ipad-sm": "(npx react-native webpack-start & ./scripts/run-build.sh --ipad-sm)",
+    "ipad-sm-standalone": "(npx react-native webpack-start & STANDALONE_NEW_DOT=true ./scripts/run-build.sh --ipad-sm)",
     "start": "npx react-native start",
```

I think we're missing the same here
```suggestion
    "start": "npx react-native webpack-start",
```

---

@adhorodyski fixed!